src/zarr/core/group.py

```diff
@@ -998,115 +1000,83 @@ async def create_array(
         name: str,
         *,
         shape: ShapeLike,
-        dtype: npt.DTypeLike = "float64",
-        fill_value: Any | None = None,
+        dtype: npt.DTypeLike,
+        chunks: ChunkCoords | Literal["auto"] = "auto",
+        shards: ChunkCoords | Literal["auto"] | None = None,
+        filters: Iterable[dict[str, JSON] | Codec] = (),
+        compressors: Iterable[dict[str, JSON] | Codec] = (),
```

Comment:
Are these supposed to be `"auto"`? Otherwise the default codecs will not be used. I found out because `tests/test_metadata/test_consolidated.py::TestConsolidated::test_consolidated` fails

Reply:
I changed that in https://github.com/zarr-developers/zarr-python/pull/2463/commits/ea3ed0e499d9ed6826c2f61d97e86c60084d285d. Feel free to revert.